src/main/java/seedu/address/logic/commands/event/ViewUpcomingEventsCommand.java

```diff
@@ -0,0 +1,96 @@
+package seedu.address.logic.commands.event;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_OPTION;
+
+import java.time.LocalDate;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.StartDateWithinTimeFramePredicate;
+
+/**
+ * Shows a list of all Events in the scheduler that occur in the next specified days.
+ */
+public class ViewUpcomingEventsCommand extends EventCommand {
+    public static final String COMMAND_OPTION = "u";
+
+    public static final String MESSAGE_INVALID_EVENT_UPCOMING_DAYS =
+            "The days provided is invalid (must be a positive integer)\n%1$s";
+    public static final String MESSAGE_MISSING_DAYS = "Days must be specified!\n%1$s";
+
+    public static final String MESSAGE_SUCCESS_MULTIPLE_EVENTS = "You have these upcoming events ";
+    public static final String MESSAGE_SUCCESS_SINGLE_EVENT = "You have 1 event ";
+    public static final String MESSAGE_SUCCESS_NO_EVENTS = "You have no upcoming events ";
+    public static final String MESSAGE_SUCCESS_TOMORROW = " tomorrow!\n";
+    public static final String MESSAGE_SUCCESS_UPCOMING_DAYS = "in the next %1$s days!";
+
```

Comment:
Do remember to add a `MESSAGE_HELP` and add it into the hash map into `EventCommandParser#getEventCommands` for it to display in the help window